spec/lib/guard/notifier_spec.rb

```diff
@@ -41,4 +41,42 @@
       end
     end
   end
+
+  describe "backward compatibility" do
+    let(:notifier) { instance_double("Notiffany::Notifier") }
```

Comment:
This line is generic for all tests in this file - you can extract it out to the top (and from the "toggle notification" tests above) - there's no need to duplicate them.